packages/appeal-reply-service-api/src/controllers/reply.js

```diff
@@ -73,35 +74,40 @@ module.exports = {
       .collection(dbId)
       .findOne({ _id: idParam })
       .then(async (originalDoc) => {
```

Comment:
Comment out rather than remove the custom logging for Horizon please.

Reply:
Re-added, and commented as requested. Sorry about that!

Reply:
It looks like some of the 'STEVE' logs are still out?

Reply:
https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/1174/files#diff-9b363b6ec9a80d94cff8dffbe57677b8cca5f9f7626f83a4048f593dbc214f76R78

and

https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/1174/files#diff-9b363b6ec9a80d94cff8dffbe57677b8cca5f9f7626f83a4048f593dbc214f76R90

Now back in play, sorry about that!